src/styles/main.scss

```diff
@@ -1,7 +1,14 @@
-@import "utils";
-@import "fonts";
-@import "typography";
+@import "./utils/vars.scss";
+@import "./utils/mixin.scss";
+@import "./utils/normalaize.scss";
```

Comment:
There is a typo in the imported file name. Rename 'normalaize.scss' to 'normalize.scss'.